quicksort/quicksort.py

```diff
@@ -0,0 +1,16 @@
+def s(l,st,end):
+    print("st,end: ", st, end)
+    if len(l) <= 1: return
+    idx = random.randint(st,end)
```

Comment:
Right before it crashes I get  the output:
`st,end:  0 -1`
so it means one of your recursive cases are set up incorrectly because clearly end should never be less than st

Reply:
this is I understand too (but code us not mine) - it was given in explanation of QS